bundles/org.openhab.persistence.dynamodb/README.md

```diff
@@ -50,6 +50,8 @@ Modify the policy accordingly if needed.
 
 **Note 2:** As a more simple alternative, one can use pre-existing policy `AmazonDynamoDBFullAccess`, although the policy grants the openHAB user wider-than-necessary permissions.
 
+**Note 3:** The Number in arn:aws:dynamodb:eu-west-1:084669220525:table/openhab must be replaced by the unique ID of your personal DB. So you must give your openhab DB User the existing Policy `AmazonDynamoDBFullAccess` that he could create the new Database. Once it is created you could click on the new openhab table and could see your uniquiqe Amazon Resource Name which you must insert in the JSON Script. After that you could revoke the `AmazonDynamoDBFullAccess` from the openhab User.
```

Comment:
BTW, would it better to have wildcard in the resource so that new table can be created?

Reply:
Wildcard in the Policy dont grant you the Right to create a new table.  You first need the Table because you need the arn to grant the openHAB User access to this Table. The Policy AmazonDynamoDBFullAccess is like a Mysql Rootuser who could create Databases. Or in DynamoDB tables.